src/core/fonts.js

```diff
@@ -133,7 +134,56 @@ function adjustWidths(properties) {
   properties.defaultWidth *= scale;
 }
 
-function adjustToUnicode(properties, builtInEncoding) {
+function adjustTrueTypeToUnicode(properties, isSymbolicFont, nameRecords) {
+  if (properties.isInternalFont) {
+    return;
+  }
+  if (properties.hasIncludedToUnicodeMap) {
+    return; // The font dictionary has a `ToUnicode` entry.
+  }
+  if (properties.hasEncoding) {
+    return; // The font dictionary has an `Encoding` entry.
+  }
+  if (properties.toUnicode instanceof IdentityToUnicodeMap) {
+    return;
+  }
+  if (!isSymbolicFont) {
+    return; // A non-symbolic font should default to `StandardEncoding`.
+  }
+  if (nameRecords.length === 0) {
+    return;
+  }
+
+  // Try to infer if the fallback encoding should really be `WinAnsiEncoding`.
+  if (properties.defaultEncoding === WinAnsiEncoding) {
```

Comment:
nit: I think it should be before the previous check.

Reply:
Well, I wrote it this way since it's not impossible to imagine that we'll need to extend this function sometime in the future to check for e.g. `MacRomanEncoding` as well. In that case, I believe that the current order makes more sense.

Also, please note that for TrueType fonts that have /ToUnicode or /Encoding data we'll break well before reaching these checks.